api/desecapi/tests/test_domains.py

```diff
@@ -270,9 +270,7 @@ def test_list_domains_owns_qname(self):
     def test_delete_my_domain(self):
         url = self.reverse("v1:domain-detail", name=self.my_domain.name)
 
-        with self.assertRequests(
-            self.requests_desec_domain_deletion(self.my_domain)
-        ):
+        with self.assertRequests(self.requests_desec_domain_deletion(self.my_domain)):
```

Comment:
This commit should be squashed into 5b177d628ffcba082d91c04ef138e90e6402a3aa

Reply:
done